app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %>
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
```

Comment:
- [ ] Please ensure that the form for registering a user matches the one provided in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_login.png), as stated in the requirement.

#### Requirement :point_down: 
![Screenshot from 2023-05-19 19-28-14](https://github.com/Chaseknis/blog_app/assets/89228743/a44ac0c5-363e-4b9c-98e9-046b4d813bdb)

### Your form so far 
![Screenshot from 2023-05-19 19-18-43](https://github.com/Chaseknis/blog_app/assets/89228743/f3b03ce8-23fa-4fbf-b818-ea8676417bf2)